tsconfig.build.json

```diff
@@ -13,7 +13,6 @@
     "sourceMap": true,
     "forceConsistentCasingInFileNames": true,
     "isolatedModules": true,
-    "exactOptionalPropertyTypes": true,
```

Comment:
Oh, I wish we could keep it. Optionally, `"noUncheckedIndexedAccess": true,` if possible.

Reply:
> Oh, I wish we could keep it. 

I was able to make `exactOptionalPropertyTypes`work (it resulted in a better DX, too, thank you!). Fixed it in [this commit](https://github.com/jotaijs/jotai-devtools/pull/30/commits/6c9cbdffa020e9f15e1e0198d18eefae0b2e4f28)

> Optionally, "noUncheckedIndexedAccess": true, if possible.

We haven't changed that, so it should [already be enabled](https://github.com/jotaijs/jotai-devtools/blob/main/tsconfig.build.json#L10) 🙌

Reply:
oops, my bad > already be enabled.